lib/repo_tools.pm

```diff
@@ -92,6 +93,13 @@ sub smt_mirror_repo {
     save_screenshot;
 }
 
+sub password_twice {
```

Comment:
mind renaming to `type_password_twice`? or something that relates more to what the function does, more than enterning the password twice...